aio/src/styles/2-modules/_resources.scss

```
@@ -11,6 +11,18 @@
 
 .resources-container {
   position: relative;
+  .group-buttons {
```

Please leave an empty line between own styles and child-element styles:

```suggestion

  .group-buttons {
```